pkg/machines/services.es6

```diff
@@ -26,10 +26,9 @@ export function spawnProcess({ cmd, args = [], stdin, failHandler }) {
 
     return spawn(cockpit.spawn(spawnArgs, { err: "message", environ: ['LC_ALL=C'] })
         .input(stdin))
-        .fail((exception, data) => {
+        .catch((exception, data) => {
```

Comment:
If we change this to `catch`, then we need to return a failed promise in the normal case.